get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,167 @@
+
+Shiny's reactive engine avoids unnecessary computations by only re-calculating the outputs whose inputs have changed, making Shiny ideal for fast data-driven applications, enabling live updates for [charts, tables](/components/#outputs), and [reports](https://quarto.org/docs/interactive/shiny/) with minimal effort.
+
+Below is a Shiny application where you have the output text react to _both_ input slider values
+and the second row of text _only_ reacts to the first slider value.
+
+```{shinylive-python}
+#| standalone: true
+#| components: [editor, viewer]
+#| layout: vertical
+#| viewerHeight: 250
+
+from shiny import reactive
+from shiny.express import ui, render, input
+
+# input slider 1
+ui.input_slider("slider1", "Slider 1", min=1, max=50, value=20)
+
+# input slider 2
+ui.input_slider("slider2", "Slider 2", min=1, max=50, value=40)
+
+# output reacts to both sliders
+@render.text
+def both_sliders_output():
+    return f"You have set Slider 1: {input.slider1()}, Slider 2: {input.slider2()}"
+
+# output that only reacts to slider 1
+@render.text
+def result():
+    return f"The value for slider 1 squared: {input.slider1()**2}."
+```
+
+## Jumpstart with templates {#templates}
+
+Below is a full Shiny application in action, complete with [reactivity](/docs/reactive-foundations.qmd) and a [user interface](/docs/ui-overview.qmd)!
+
+::::: {.column-screen-inset}
+::: {.hello-output}
+<iframe
+  src="https://gallery.shinyapps.io/template-dashboard/"
+  frameborder="0"
+  width="100%"
+  class="mb-0 card hello-output-iframe"
+></iframe>
+
+<style>
+  .hello-output-iframe {
+    height: 700px;
+  }
+@media only screen and (max-width: 611px) {
+  .hello-output-iframe {
+    height: 2125px;
+  }
+}
+</style>
+:::
+:::::
+
+Hit the ground running with one of our [starter templates](/templates/), like the app above, by using `shiny create`.
+
+There are templates for common use cases.
+For example, [data dashboards](templates/dashboard/), [applications](/templates/database-explorer/), [streaming](/templates/monitor-database/) updates, or [data entry](/templates/survey/).
+
+Use the terminal command below to [create and run](create-run.qmd) your first application from a [template](/templates/).
+
+```bash
+shiny create --template dashboard
+```
+
+
+## Extensibility {#extensible}
+
+While Shiny includes everything you need to build an app, is built on a foundation of web standards, making it highly [extensible](/docs/).
+
+If you're versed in web programming, you can also use Shiny to create [custom components](/docs/custom-component-one-off.qmd) that leverage your favorite JavaScript framework from Python.
+
+If you need a [custom component](/docs/custom-components-pkg) or [user interface](/docs/ui-customize), you can incrementally integrate [JavaScript](/docs/custom-component-one-off), [HTML](/docs/ui-html.html), or even WebAssembly without having to learn complicated build tooling and frameworks.
+
+The Python inputs return regular HTML:
```

Comment:
Combine with first paragraph and mention "In fact, Shiny for Python is built on standard HTML..."